readme.md

```diff
@@ -1,56 +1,64 @@
+3. Upload the Gravity Forms Paystation (3 party hosted) plugin to your /wp-content/plugins/ directory
 4. Activate Gravity Forms Paystation (3 party hosted) via Plugins menu in WordPress.
 
-## Using Gravity Forms Paystation
+### Create a gravity form:
+
+1. To create a gravity form, go to the admin dashboard, see the gravity form plugin which you have installed, and click add new form.
+2. Create your form according to your requirements, please make sure that you should include the form field from the payment details tab to ensure that the our plugin is working correctly. 
+3. Select your form from the dropdown at the top of the page and click save.
+4. If you want custom field mappings to Paystation data, you can map your form fields to Paystation transaction fields (Merchant Reference, Customer details, Order details), be aware
+   a form field can only be mapped to one Paystation transaction field at a time.
+
+## Using Gravity Forms Paystation:
 
 Before you can use the plugin to make payments, you need to create an account with Paystation.
 To do this please see visit the [Paystation](https://www2.paystation.co.nz/) website.
 
 Then you will need to enter some information in the plugin settings
 
 1. Select 'Plugins' from the WordPress Admin menu.
-2. Click the 'Settings' link for the Gravity Forms Paystation (3 party hosted) plugin.
-3. Enter your Paystation Id in the Paystation Id field.
-4. Enter your Paystation Gateway Id in to the Paystation Gateway Id field.
-5. Enter a string of letters and numbers for the Security Hash (8 to 20 characters is fine).
-6. Save the settings by clicking the 'Save Settings' button.
-7. Email info@paystation.co.nz letting us know the following: that you are using the Paystation Gravity forms plugin, the url to your wordpress site, and what your Security Hash is.
-
-### Building a Gravity Form with Credit Card Payments
-
-1. Create a new Gravity form
-1. Add one or more Product fields or a Total field to your form. The plugin will automatically detect the values assigned to these pricing fields
-3. Return to Paystation (3 party) menu where you setup your Paystation settings above and click 'Setup Paystation Feed'
-4. Click 'Add New' feed at the top of the page
-5. Select your form from the dropdown at the top of the page and save
-6. If you want custom field mappings to Paystation data, you can map your form fields to Paystation transaction fields (Merchant Reference, Customer details, Order details), be aware
-   a form field can only be mapped to one Paystation transaction field at a time.
+2. Select the Gravity Forms plugin, the 3 party setting will be under the gravity forms plugin.
+3. And then choose the Paystation (3 party hosted).
+4. Enter your Paystation Id in the Paystation Id field, Gateway ID in the gateway Id field. 
+5. Enter a string of letters and numbers for the Security Hash (8 to 20 characters is fine). 
+6. Then choose 'No' in the test mode field. 
```

Comment:
Is this for when you are ready to go live? Perhaps this can be a change that goes at the end once the merchant is ready for production action

Reply:
I've updated this section to allow merchants to toggle the test mode on or off at their end

Reply:
Yes, when ready to go live